SConstruct

```diff
@@ -239,7 +239,14 @@ opts.Add(BoolVariable("disable_physics_3d", "Disable 3D physics nodes and server
 opts.Add(BoolVariable("disable_navigation_2d", "Disable 2D navigation features", False))
 opts.Add(BoolVariable("disable_navigation_3d", "Disable 3D navigation features", False))
 opts.Add(BoolVariable("disable_xr", "Disable XR nodes and server", False))
-opts.Add(BoolVariable("disable_overrides", "Disable project settings overrides and related CLI arguments", False))
+opts.Add(BoolVariable("disable_overrides", "Disable project settings overrides (override.cfg)", False))
+opts.Add(
+    BoolVariable(
+        "disable_path_overrides",
+        "Disable CLI arguments to override project path/main pack/scene and run scripts (export template only)",
+        True,
```

Comment:
It feels a bit weird to me that `disable_path_overrides` is silently ignored when building an editor build. But I'm not sure if it's important to fix this.

Reply:
The main solution in my mind would be to have it 'unspecified' by default, and if it's specified it overrides the defaults. But I'm not sure if we do this for any other options, plus the editor wouldn't function as expected with the option explicitly enabled anyway. So it's probably fine to keep it as is.